packages/components/tailwind.config.js

```diff
@@ -3,7 +3,7 @@
 const { Preset } = require("@kaizen/tailwind")
 
 module.exports = {
-  content: ["./src/**/*.tsx"],
+  content: ["./src/**/*.{stories.tsx,mdx}"],
```

Comment:
Perhaps we don't even need this file at all and just rely on the global one to cater for all the stories (haven't checked, unsure if true)?

Reply:
I axed it with no issues :) looks like we're in the clear for `@kaizen/components`